src/components/my-view2.js

```diff
@@ -32,20 +32,20 @@ class MyView2 extends connect(store)(PageViewElement) {
       ${SharedStyles}
       <section>
         <h2>Redux example: simple counter</h2>
-        <div class="circle">${props._clicks}</div>
+        <div class="circle">${props._value}</div>
         <p>This page contains a reusable <code>&lt;counter-element&gt;</code>. The
         element is not built in a Redux-y way (you can think of it as being a
         third-party element you got from someone else), but this page is connected to the
         Redux store. When the element updates its counter, this page updates the values
-        in the Redux store, and you can see the total number of clicks reflected in
+        in the Redux store, and you can see the current value of the counter reflected in
         the bubble above.</p>
         <br><br>
       </section>
       <section>
         <p>
           <counter-element value="${props._value}" clicks="${props._clicks}"
-              on-counter-incremented="${() => store.dispatch(increment())}"
-              on-counter-decremented="${() => store.dispatch(decrement())}">
+              on-counter-decremented="${() => store.dispatch(decrement())}"
```

Comment:
this ordering does not matter (the order of attributes in HTML is irrelevant), so i don't think this change is needed

Reply:
I know it's functionally relevant, just a matter of consistency. In any case, since we are not changing the order, this shouldn't matter.